sass/edx/_variables.scss

```diff
@@ -245,19 +252,19 @@ $h6-font-size: 1rem !default;
 //
 //$small-font-size: 80% !default;
 //
-//$text-muted: $gray-600 !default;
+//$text-muted: theme-color("secondary") !default;
```

Comment:
Note that you probably shouldn't change the commented out line, as they are intended to indicate what the default is.

Reply:
I changed a lot of the comments to use the theme colors, thinking it would keep consistency if they were ever to remove the comments. Do you think all them should go back to what they were previously?

Reply:
Hmm, interesting. I'd been thinking of the commented out lines showing what Bootstrap was using, but maybe that isn't necessary. I'm fine either way.